internal/pkg/describe/rd_web_service.go

```diff
@@ -126,6 +169,7 @@ type rdWebSvcDesc struct {
 	Routes                  []*WebServiceRoute      `json:"routes"`
 	Variables               envVars                 `json:"variables"`
 	Resources               deployedSvcResources    `json:"resources,omitempty"`
+	Observability           []Observability         `json:"observability,omitempty"`
```

Comment:
Should this be
```suggestion
	Observabilities           []Observability         `json:"observabilities,omitempty"`
```

Reply:
It's just one `observability` configuration - it's only a list because there are several environments (sort of like a map but in the form of a list if I'm to say). I feel like singular is better suited in this particular case 🤔